tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "experimentalDecorators": true,
     "importsNotUsedAsValues":"remove",
     "target": "es2018",
-    "sourceMap": false,
+    "sourceMap": true,
```

Comment:
I moved the sourceMap false to the tsconfig.build.json. This way, the dist still doesn't have the .map's, but the debug experience is better